create_crio_sysext.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export ARCH="${ARCH-x86-64}"
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME"
+  echo "The script will download the cri-o release binaries (e.g., for v1.28.4) and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "To use arm64 pass 'ARCH=arm64' as environment variable (current value is '${ARCH}')."
+  echo "CNI files are not automatically installed systemwide, but are present in /usr/share/crio/cni"
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+
+# The github release uses different arch identifiers (not the same as in the other scripts here),
+# we map them here and rely on bake.sh to map them back to what systemd expects
+if [ "${ARCH}" = "x86_64" ] || [ "${ARCH}" = "x86-64" ]; then
+  ARCH="amd64"
+elif [ "${ARCH}" = "aarch64" ]; then
+  ARCH="arm64"
+fi
+
+curl -o "cri-o.${ARCH}.v${VERSION}.tar.gz" -fsSL "https://storage.googleapis.com/cri-o/artifacts/cri-o.${ARCH}.v${VERSION}.tar.gz" 
+rm -rf "${SYSEXTNAME}"
+mkdir -p "${SYSEXTNAME}" "${SYSEXTNAME}/tmp"
+tar --force-local -xf cri-o.${ARCH}.v${VERSION}.tar.gz -C "${SYSEXTNAME}/tmp"
+cd ${SYSEXTNAME}/tmp/cri-o/
+sed -i '/^sed -i.*DESTDIR/d' install # removes sed replacements from install script to keep the default location (/usr) in the base config file
```

Comment:
This smells weird for me. Can't you set it to / instead of deleting it and it just works?

Reply:
@rata  Are you talking about the line 34 ? If i do not delete those lines from the install script the final 10-crio.conf will be broken due to this:
```
# Update the DESTDIR in the CRI-O configuration
sed -i 's;/usr/bin;'"$DESTDIR$BINDIR"';g' "$DESTDIR$ETC_CRIO_DIR/crio.conf.d/10-crio.conf"
sed -i 's;/etc/crio;'"$DESTDIR$ETC_CRIO_DIR"';g' "$DESTDIR$ETC_CRIO_DIR/crio.conf.d/10-crio.conf"

```

Reply:
Yes, but if you set DESTDIR to `/`, doesn't it all work? All those things you are shoing should expand correctly if you set it to "/".

Reply:
And if you leave it empty probably too.

Reply:
But if I set to / or leave it empty the script will try to install to / inside the machine running the install script and i need it to install to the extension dir.

Reply:
Ohhh, sorry then. I'm surprised you can't set it to the right value, but anyways is not a big issue. Ignore this, sorry :)